src/ontology/fbbt-edit.obo

```diff
@@ -146106,6 +146106,96 @@ relationship: develops_from FBbt:00059159 ! cyst cell of female germline 8-cell
 property_value: http://purl.org/dc/terms/contributor http://orcid.org/0000-0002-6095-8718
 property_value: http://purl.org/dc/terms/date 2022-08-30T16:50:20Z xsd:dateTime
 
+[Term]
+id: FBbt:00059161
+name: leg disc epithelium proper
+def: "." [FBC:Autogenerated]
+subset: cur
+intersection_of: FBbt:00007029 ! disc epithelium proper
+intersection_of: part_of FBbt:00001780 ! ventral thoracic disc
```

Comment:
thought you were going to rename to 'leg disc'(?)

Reply:
Indeed, forgot that one… 🤦